store/repository.go

```diff
@@ -385,6 +304,7 @@ func (s *Store) createRepositoryImpl(ctx context.Context, tx *Tx, create *api.Re
 			base_directory,
 			file_path_template,
 			schema_path_template,
+			sheet_path_template,
```

Comment:
@ecmadao do we also miss the enable_sql_review_ci field upon creation?

Reply:
After taking a look, enable_sql_review_ci is set by patch. Though, we can use a default FALSE for creating the repository entity.

Reply:
Right, we’ve changed our UX workflow and will patch the repo to enable the CI after creation.
Agree with @d-bytebase, we can manually insert the `FALSE` and add the comment to avoid confusion.

Reply:
Per chat, @ecmadao will make another PR for this.